front/pages/api/registry/[type]/lookup.ts

```diff
@@ -180,28 +180,37 @@ async function handleDataSourceView(
     auth,
     dataSourceViewId
   );
-  if (!dataSourceView) {
+
+  // This check is meant to block access to "conversations" space through a
+  // datasource block in a dust app, which could lead to data leaks, see related PR
```

Comment:
please add link to PR

Reply:
had removed it because we can get it from IDE / git blame easily on the comment
but ok to re-add :+1: